### DISCUSSION TITLE
ait-wiki/blocks/engine-core/

### DISCUSSION THREAD

**Discussion by giscus** [amblelabs/ait-wiki#11] [General]
# ait-wiki/blocks/engine-core/

The Engine Core is the main “heart” of The TARDIS used to power your whole TARDIS.
How do I get the Engine Core? How to activate the core? You can activate the Engine Core by placing it in the middle of a 3x3x3 water area like how you would activate a conduit. This will activate the Engine Core allowing The TARDIS to recive power from the TARDIS Star, which will spawn under The TARDIS Interior

https://pavatus.github.io/ait-wiki/blocks/engine-core/

**Comment by 0LordVoid0** [comment]
I can't seem to figure out how to activate it. Is there a vid I can watch? It seems to be different than described in the instructions. probably do to an update. Can you guys update the instructions?

**Comment by Addi3** [comment]
if your playing a version that is 1.1.0 + the engine has been massively changed, an updated wiki is being worked on but isnt public

**Reply by 0LordVoid0** [reply]
Ok, thanks

**Reply by Addi3** [reply]
there is a somewhat of a guide here https://discord.com/channels/1213989169878274068/1253351515305807914/1318997700292907163

**Reply by 0LordVoid0** [reply]
I can't access this. It says there is nothing.

**Reply by Addi3** [reply]
whats your discord, ill @ you in general

**Reply by 0LordVoid0** [reply]
bill_cipher0774

**Reply by Addi3** [reply]
are you in the ait discord, you dont show up

**Reply by 0LordVoid0** [reply]
Thank you

**Reply by Addi3** [reply]
np

**Comment by Vlada3333** [comment]
Interesting.

**Comment by HydronBlaster** [comment]
What do I do if it's green?

**Reply by Mansarde** [reply]
That happens when the alarm is on or during a flight event.

**Comment by Walshy99** [comment]
changed the console room to the coral one. placed and replaced the engine countless times but it doesn't grow or glow. just looks brown and dead with the blue galifreyan particle affects coming off it.
 what do i do to fix it?

**Comment by greerjermaiha-source** [comment]
so are their specific places we need to put an engine because when i place an enine im my tardis it isnt powering on
